eventActions/cotwActions.js

```diff
@@ -41,10 +41,12 @@ class cotwActions {
 				});
 
 				winner.addRole(cotwRole);
```

Comment:
does it remove role from the previous person automatically? since it's not part of the PR i'll approve- looks good!